From: =?utf-8?q?Martin_Stegh=C3=B6fer?= <martin@steghoefer.eu>
Date: Fri, 12 Dec 2014 23:17:02 +0100
Subject: Use translations in oggdec

The "oggdec" executable uses the "_" (underscore) macro for
internationalization and has translations of the relevant strings
available in the .po files, but nevertheless outputs everything
in English. This was due to a lack of initialization of the
internationalization modules in that executable.

Bug-Debian: https://bugs.debian.org/772976
Forwarded: https://trac.xiph.org/ticket/2149
---
 oggdec/oggdec.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/oggdec/oggdec.c b/oggdec/oggdec.c
index 16f87ac..3f2ae7b 100644
--- a/oggdec/oggdec.c
+++ b/oggdec/oggdec.c
@@ -16,6 +16,7 @@
 #include <getopt.h>
 #include <errno.h>
 #include <string.h>
+#include <locale.h>
 
 #if defined(_WIN32) || defined(__EMX__) || defined(__WATCOMC__)
 #include <fcntl.h>
@@ -364,6 +365,10 @@ int main(int argc, char **argv)
 {
     int i;
 
+    setlocale(LC_ALL, "");
+    bindtextdomain(PACKAGE, LOCALEDIR);
+    textdomain(PACKAGE);
+
     if(argc == 1) {
         usage();
         return 1;
